src/segmentation.rs

```diff
@@ -7,14 +7,23 @@
 // Media Patent License 1.0 was not distributed with this source code in the
 // PATENTS file, you can obtain it at www.aomedia.org/license/patent.
 
+use std::cmp::Ordering;
+
 use crate::context::*;
 use crate::header::PRIMARY_REF_NONE;
 use crate::partition::BlockSize;
+use crate::rdo::spatiotemporal_scale;
+use crate::rdo::DistortionScale;
 use crate::tiling::TileStateMut;
 use crate::util::Pixel;
 use crate::FrameInvariants;
 use crate::FrameState;
 
+pub const MAX_SEGMENTS: usize = 8;
+
+// This value was obtained through testing over multiple runs in AWCY.
+const BASE_AQ_MULT: f64 = -9.0;
```

Comment:
Is the idea that an AQ strength of 1 is optimal?

Reply:
Yes, for the sake of user experience. A typical user would assume that the default strength should be "1.0", and then we try to make the default as optimal as possible.